flask_rebar/rebar.py

```diff
@@ -146,11 +146,19 @@ def wrapped(*args, **kwargs):
         if not response_body_schema:
             return rv
 
-        data, status_code, headers = _unpack_view_func_return_value(rv)
+        if isinstance(rv, current_app.response_class):
+            schema = response_body_schema[rv.status_code]
```

Comment:
```suggestion
            schema = response_body_schema.get(rv.status_code, None)
```
I know this is just duplicating the logic on 159, but I think it's worth noting I see issues with this approach.
IMO, we should either throw an explicit "invalid status" error if the status is not one defined in the response body schema, or return a default; throwing a KeyError here has the potential to cause unnecessary confusion.